Source/cmake/OptionsCommon.cmake

```diff
@@ -153,20 +153,22 @@ if (USE_THIN_ARCHIVES)
     set(CMAKE_C_ARCHIVE_APPEND "<CMAKE_AR> rT <TARGET> <LINK_FLAGS> <OBJECTS>")
 endif ()
 
-set(ENABLE_DEBUG_FISSION_DEFAULT OFF)
-check_cxx_compiler_flag(-gsplit-dwarf CXX_COMPILER_SUPPORTS_GSPLIT_DWARF)
-if (CXX_COMPILER_SUPPORTS_GSPLIT_DWARF AND LD_SUPPORTS_SPLIT_DEBUG)
-  set(ENABLE_DEBUG_FISSION_DEFAULT ON)
-endif ()
+if (NOT (CMAKE_GENERATOR MATCHES "Visual Studio"))
```

Comment:
Wouldn't it be better to check `CMAKE_CXX_COMPILER_ID NOT STREQUAL MSVC`? I suppose nobody would use the Visual Studio generator with a different compiler than MSVC, but one can reasonable use Make/Ninja with MSVC (AFAIU).

Reply:
Thanks for the clatification, that helps 👌🏼